src/api/api.ts

```diff
@@ -20,3 +20,46 @@ export async function getAssetList({
 
   return response.items;
 }
+
+export async function retrieveAsset(assetId: number) {
+  return await Assets.retrieve(assetId);
+}
+
+export async function getAssetListDescendants(
+  assetId: number,
+  query: { [name: string]: any }
+) {
+  const response = await Assets.listDescendants(assetId, query);
+
+  if (!response.items || response.items.length < 1) {
+    return [];
+  }
+
+  return response.items;
+}
+
+export async function getAssetEvent(query: {
+  assetId: number;
+  limit: number;
+}): Promise<Event[]> {
+  const response = await Events.list(query);
+
+  if (response.items && response.items.length > 0) {
+    return response.items;
+  }
+
+  return [];
+}
+
+export async function getAssetFiles(query: {
+  assetId: number;
+  limit: number;
+}): Promise<File[]> {
+  const response = await Files.list(query);
+
+  if (response.items && response.items.length > 0) {
```

Comment:
I think `response.items` will always be an array. So think this if-statement is unnecessary

Reply:
fixed